src/main/webapp/script.js

```diff
@@ -33,7 +34,13 @@ function displayLocation(position) {
 }
 
 function displayError() {
-  // TODO: Write error to user interface
   console.log('Geolocation not enabled');
   alert(alertMessage);
+}
+
+function getZip() {
```

Comment:
*getZipCode(). There's no real need to short hand it.

Reply:
Got it - it's changed